examples/analog/AO_multipleChannels.py

```diff
@@ -69,10 +76,44 @@
     for i in range(0, 20):
         # define a list of floating-point numbers
         input_value = [1.0, 2.5]
-        # specifies the sampling frequency, in hertz, of the output signal.
+        # specify the sampling frequency, in hertz, of the output signal.
         sample_rate = 1000
         # write 1.0 and 2.5 to both AO0 and AO1 on bank A
         AO_multiple_channels.write(input_value, sample_rate)
 
         # add a short delay before writing the next data point
         time.sleep(0.001)
+
+##############################################################################
+# Section 3: Multiple Points (continuous)
+# Use the start_continuous_mode, write, and stop_continuous_mode functions to
+# write multiple points of data to the channels. The hardware continuously
+# generates samples for a channel until the generation is stopped.
+##############################################################################
+# configure the AO channels
+with AnalogOutput({'bank': bank,     # define first channel: AO0
+                   'channel': channel0},
+                  {'bank': bank,     # define second channel: AO1
+                   'channel': channel1}) as AO_multiple_channels:
+    # define a list of floating-point numbers, the input value should be
+    # defined in the format: [[values_for_first_channel],
+    # [values_for_second_channel], ...]
+    input_value = [[1.0, 2.0, 3.0, 4.0, 5.0], [5.0, 4.0, 3.0, 2.0, 1.0]]
+    # specify the sampling frequency, in hertz, of the output signal
```

Comment:
```suggestion
    # specify the sampling frequency, in Hz, of the output signal
```

Reply:
@ling-wang

Reply:
I suggest using "hertz" because we are explaining the unit of sampling frequency here. We'd better spell out this word. For other cases, for example, 500 Hz, or in equations, we use the symbol "Hz".